modules/optflow/src/rlofflow.cpp

```diff
@@ -203,7 +203,10 @@ class DenseOpticalFlowRLOFImpl : public DenseRLOFOpticalFlow
             filtered_prevPoints = prevPoints;
             filtered_currPoints = currPoints;
         }
-
+        // Interpolators below expect non empty matches
+        if (filtered_prevPoints.empty())
+            return;
```

Comment:
/cc @tsenst Could you please take a look on this change? Should we "reset" some fields/parameters to return consistent result?

Reply:
I ammended the commit with correct author, i think this should be fixed

Reply:
I can take a look during the next days

Reply:
@MrKepzie thank you for this bugfix :+1: . Here the optical flow could not be computed because of the algorithm behaviour. It would be great if it would be possible to indicate this. Unfortunately the DenseOpticalFlow class has no interface for the error case. Maybe throwing an exception would be the correct way. If not the flow should be at least set to 0 ( e.g. flow.setTo(0) ). In both cases please update the documentation.

Reply:
`CV_Error(Error::StsNoConv, ...)` may be used to indicate issue (throwing an exception).

Reply:
I think it's better to return a vector field with 0 everywhere is this is not really an error but just 2 images without any correspondences, I'm going to ammend commit and do as suggested

Reply:
And please add a note in the docfile that when no correspondences could be found for interpolation the optical flow field would be 0

Reply:
Just updated the commit